KP-Cratefiller/KPCF/fnc/fn_manageAceActions.sqf

```diff
@@ -0,0 +1,20 @@
+/*
+    Killah Potatoes Cratefiller v1.1.0
+
+    Author: Dubjunk - https://github.com/KillahPotatoes
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+    Adds actions to the cratefiller objects.
+
+    Parameter(s):
+    0 : OBJECT - defines the object to apply the action
+
+    Returns:
+    NONE
+*/
+
+params ["_cfBase"];
```

Comment:
I would suggest to be as precise as you can be concerning parameters etc.
This way it's much easier to debug later on, if it happens that something or someone calls your function with not intended or catched datatypes. (suggestion applies to all params usages, not just this particular one)
```suggestion
params [
    ["_cfBase", objNull, [objNull]]
];
```